assets/js/blocks/classic-template/index.tsx

```diff
@@ -60,6 +81,13 @@ const Edit = ( {
 		[ attributes.align, attributes.template, setAttributes ]
 	);
 
+	// Blockification is possible for the WP version 6.1 and above,
+	// which are the versions the Products block supports.
+	const isBlockificationPossible = isWpVersion( '6.1', '>=' );
```

Comment:
For now, `isBlockificationPossible` should be false when in the Single Product template:

![imatge](https://user-images.githubusercontent.com/3616980/214588943-886888b9-3ac2-45a7-a7c3-dcaaaed21ca8.png)

It should also have a different description and eventually a different button label.

Reply:
Great catch!

1. I extended the testing steps to cover the Single Product template (expected: description which doesn't indicate the update and no button)
2. I made the blockification more generic so it's prepared for Single Product template blockification. For the Single Product template it will be a matter of changing providing proper config [in here](https://github.com/woocommerce/woocommerce-blocks/pull/8248/files#diff-49cf8439043f78ee9f68b6374ef821be0b59c5804fec85e9aecf528d7fe9bcecR50) in the same manner as the product archive. If that change is ok and will be merged, I'll update the issue regarding Single Product template to let the developers know.